documentation/Deployment-Guide.md

```diff
@@ -0,0 +1,1136 @@
+
+![alt text](https://github.com/sysgain/SecurityAndSurveillance/blob/master/images/d54.png)
+
+11.	Click **appcert** certificate there you can see the thumbprint of certificate as highlighted in the following figure. Note down the thumbprint.
+
+![alt text](https://github.com/sysgain/SecurityAndSurveillance/blob/master/images/d55.png)
+
+## 4 ARM Template Input Parameters
+
+In the parameters section of the template, specify the values as inputs when deploying the ARM template. These parameter values enable you to customize the deployment by providing values that are tailored for your environment (such as dev, Staging and production). 
+
+
+| **Parameter Name**  | **Description**     | **Allowed Values**    | **Default Values**   |                                                                                                            
+| -------------       | -------------       | -----------------     | ------------         |
+| **Solution Type**   | Choose the solution deployment type from the drop down, for more information about the solution deployment type navigate to https://github.com/sysgain/SecurityAndSurveillance/blob/master/documentation/Deployment-Guide.md      | Basic, Standard, Premium | Basic solution|
+| **deploymentPrefix**   | A unique string for the deployment with maximum 15 characters which would be used to provision the resources   |         |            |
+| **cognitive Services Location**   | Choose location for CognitiveServices, for more information about the Data Lake Store navigate to https://github.com/sysgain/SecurityAndSurveillance/blob/master/documentation/Deployment-Guide.md   | eastus, eastus2, westus, westus2, southcentralus, westcentralus, northeurope, westeurope   | eastus |
+| **OMS Location**   | Choose location for OMS Log Analytics to deploy. For more information,see https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-overview   | australiasoutheast, canadacentral, centralindia, eastus, japaneast, southeastasia, uksouth, westeurope  | eastus  |
+| **App Insights Location**  | specify the region for application insights   | eastus, northeurope, southcentralus, southeastasia, westeurope, westus2 | westus2 |
+| **Location DR**  | Choose locationDR which is different from the resource group region,if you have selected standard (or) premium option in the solution type inputparameter, for more information about the locationDr navigate to https://github.com/sysgain/SecurityAndSurveillance/blob/master/documentation/Deployment-Guide.md   | NA, EastUS2, CentralUS, WestUS2, WestCentralUS, CanadaCentral, CanadaEast, AustraliaEast, AustraliaSouthEast, CentralIndia,  SouthIndia, EastAsia, SouthEastAsia, JapanWest, JapanEast, KoreaCentral,    KoreaSouth, UKSouth, UKWest | NA |
+| **Traffic Manager Name**    | Please provide the name of the trafficmanager certficate custome domain name   |    | NA |
```

Comment:
Typo "Custome" should be "custom"

Reply:
We have updated the above comment, please review and close it.